gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -95,8 +98,10 @@
   private BottomSheetBehavior<View> bottomSheetBehavior;
   private ProjectSelectorDialogFragment projectSelectorDialogFragment;
   private ProjectSelectorViewModel projectSelectorViewModel;
+  private FeatureSelectorViewModel featureSelectorViewModel;
```

Comment:
To make the purpose of this dialog clearer, would it be better to call it something like `FeatureDisambiguationDialog[ViewModel|Fragment` etc? It's wordy, but might be more self-documenting. Wdyt?

Reply:
I see your point, but it also feels unnecessarily narrow to me. I think it'd be ok to keep it more general since I think `FeatureSelector` still communicates what it is and it could also fit future scenarios (e.g. maybe we want to trigger it for some other future flow, like viewing a list of all features).